sklego/meta.py

```diff
@@ -54,6 +54,47 @@ def __init__(self, estimator, groups, use_fallback=True):
         self.groups = groups
         self.use_fallback = use_fallback
 
+    def __check_group_cols_exist(self, X):
+        try:
```

Comment:
checking the type of `X` is probably better to do via `isinstance` instead waiting for an `AttributeError`.

Reply:
I agree that in this case the intent is clearer with an isinstance check.